src/components/send-button/index.js

```diff
@@ -82,22 +106,69 @@ export default compose( [
 			publishStatus = 'publish';
 		}
 
-		const onClick = () => {
+		const triggerCampaignSend = () => {
 			editPost( { status: publishStatus } );
 			savePost();
 		};
 
+		const [ modalVisible, setModalVisible ] = useState( false );
+
 		return (
-			<Button
-				className="editor-post-publish-button"
-				isBusy={ isSaving && 'publish' === status }
-				isPrimary
-				isLarge
-				onClick={ onClick }
-				disabled={ ! isButtonEnabled }
-			>
-				{ label }
-			</Button>
+			<Fragment>
+				<Button
+					className="editor-post-publish-button"
+					isBusy={ isSaving && 'publish' === status }
+					isPrimary
+					isLarge
+					onClick={ () => setModalVisible( true ) }
+					disabled={ ! isButtonEnabled }
+				>
+					{ label }
+				</Button>
+				{ modalVisible && (
+					<Modal
+						className="newspack-newsletters__modal"
+						title={ __( 'Send your newsletter?', 'newspack-newsletters' ) }
+						onRequestClose={ () => setModalVisible( false ) }
+					>
+						<p>
+							{ __( "You're about to send a newsletter to:", 'newspack-newsletters' ) }
+							<br />
+							<b>{ listData.name }</b>
+							<br />
+							{ listData.groupName && (
+								<Fragment>
+									{ __( 'Group:', 'newspack-newsletters' ) } <b>{ listData.groupName }</b>
+									<br />
+								</Fragment>
+							) }
+							<b>
```

Comment:
```suggestion
							<strong>
```

Reply:
done in acf67a54d4e759aaaa08eb65f0b5c94ca169475f